### ISSUE TITLE
Issue with parens

### ISSUE THREAD

**Issue by x8x** [ThisIsManta/vscode-stylus-supremacy#17]
given this:
```stylus
fun(n)
  bottom (n - 1) * 2px
```
if I format above, I get:
```stylus
fun(n)
  bottom ((n - 1)) * 2px
```
one more format and I get:
```stylus
fun(n)
  bottom (((n - 1))) * 2px
```
version 2.5.2

**Comment by ThisIsManta** [comment]
Thank you for reporting the issue. I'll work on it by the end of today.

**Comment by ThisIsManta** [comment]
Hello @x8x,

The problem must be solved at version 2.5.4.

Cheers!

**Comment by x8x** [comment]
Perfect! Thank you and have a great day! :-)
